app/src/main/res/layout-land/fragment_welcome_step1.xml

```diff
@@ -14,89 +14,99 @@
     android:orientation="vertical">
 
     <LinearLayout
-      android:layout_width="wrap_content"
```

Comment:
I didn't understand why there are so many changes in this layout. :) I try to understand them but I couldn't. Can you describe? Thank you.